test/integration/main_test.go

```diff
@@ -125,14 +125,36 @@ func TestMappingEventHandling(t *testing.T) {
 		wantProcessErrSubStr string
 	}{
 		{
-			name:              "mapping_success_event",
-			resourceName:      fmt.Sprintf("//storage.googleapis.com/%s", cfg.GCSStaticBucket),
+			name:              "mapping_success_event_ci",
+			resourceName:      fmt.Sprintf("//storage.googleapis.com/%s", cfg.GCSStaticBucketCI),
 			bigqueryTable:     cfg.MappingTableID,
 			wantCAISProcessed: true,
 			wantResourceMapping: &v1alpha1.ResourceMapping{
 				Resource: &v1alpha1.Resource{
 					Provider: "gcp",
-					Name:     fmt.Sprintf("//storage.googleapis.com/%s", cfg.GCSStaticBucket),
+					Name:     fmt.Sprintf("//storage.googleapis.com/%s", cfg.GCSStaticBucketCI),
+				},
+				Contacts: &v1alpha1.Contacts{Email: []string{"group@example.com"}},
+			},
+			wantGithubSource: &v1alpha1.GitHubSource{
+				RepoName:                   testGithubRepoValue,
+				Commit:                     testGithubCommitValue,
+				Workflow:                   testWorkflowValue,
+				WorkflowSha:                testWorkflowShaValue,
+				WorkflowTriggeredTimestamp: testParseTime(t, testWorkflowTriggeredTimeValue),
+				WorkflowRunId:              testWorkflowRunID,
+				WorkflowRunAttempt:         1,
+			},
+		},
+		{
+			name:              "mapping_success_event_dev",
```

Comment:
I considered about combining these two test into one that only obtain Bucket info from pmap-dev project as pmap-dev and pmap-ci are under the same folder, and the default scope now is `folders`. But I thought having two tests here can help us make sure we can not only get resource info from the same project, but also from other project under the same folder.
Please let me know if you think this is redundant.

Reply:
I think it is unnecessary when your scope is folder, cause getting the resource info in a scope is Asset Inventory's job and asset inventory should make sure of that. Do we have a non-GCS resource test?

Reply:
We can add another test that asset inventory search fails, for example a GCS with wrong resource scope. This could provide good test coverage for custom scope, and we don't need to create resource in dev.